### ISSUE TITLE
[Website] Python API References redirect to 404 (1.6.0 Release) 

### ISSUE THREAD

**Issue by barry-jin** [apache/incubator-mxnet#18897]
## Description
Some hyperlinks of tutorials in [python API/mxnet.gluon (1.6.0 Release)](https://mxnet.apache.org/versions/1.6/api/python/docs/api/gluon/index.html) leads to 404, Including Gluon Guide, gluon.Parameter, gluon.Trainer, gluon.model_zoo.vision. 

### Error Message
404 Page

## To Reproduce
Try to access these contents through [master branch](https://mxnet.apache.org/api/python/docs/api/gluon/index.html). 

### Steps to reproduce
(Paste the commands you ran that produced the error.)

1.
2.

## What have you tried to solve it?

1.
2.

## Environment

We recommend using our script for collecting the diagnositc information. Run the following command and paste the outputs below:
```
curl --retry 10 -s https://raw.githubusercontent.com/dmlc/gluon-nlp/master/tools/diagnose.py | python

# paste outputs here
```

**Comment by leezu** [comment]
cc @ys2843

**Comment by ys2843** [comment]
These broken links were only fixed in master branch, need to backport to v1.6.

**Comment by ys2843** [comment]
Fixed the broken links on gluon index page v1.6 website, please verify. 
link: https://mxnet.apache.org/versions/1.6/api/python/docs/api/gluon/index.html
